### PR TITLE
Adding jumpstart documentation and scripts for Platform9 Managed Kubernetes cluster to connect with Azure Arc

### PR DESCRIPTION
**Pull request by deepuks**
This PR has the documentation and jumpstart scripts specific to Platform9 Managed Kubernetes.

Upstream issue tracker : https://github.com/microsoft/azure_arc/issues/796

Thanks.

### CODE REVIEW

**Comment by deepuks** [comment]
Lior, I have made the changes. Sending the PR today in next few hours.

**Comment by deepuks** [comment]
Hello Lior,

Could you please check my recent commit, I have made the changes from earlier review. Please let me know if this looks good or any changes required further.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+title: "Platform9 Managed BareOS Kubernetes (PMK) cluster"
+linkTitle: "Platform9 Managed BareOS Kubernetes (PMK) cluster"
+weight: 1
+description: >
+---
+
+## Deploy a Platform9 Managed Kubernetes cluster and connect it to Azure Arc
+
+- The following document will guide on how to deploy a Kubernetes cluster with [Platform9 Managed Kubernetes (PMK)](https://platform9.com/managed-kubernetes/) and have it as a connected Azure Arc Kubernetes resource.
+- With PMK, you can have your clusters deployed on-premises, in public clouds or at the edge. In this document, we'll explain the steps on how to create an **on-premise [BareOS](https://platform9.com/docs/kubernetes/bareos-what-is-bareos) cluster** using **PMK** and connect it to **Microsoft Azure Arc**.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:12]:
```suggestion
The following document will guide on how to deploy a Kubernetes cluster with [Platform9 Managed Kubernetes (PMK)](https://platform9.com/managed-kubernetes/) and have it as a connected Azure Arc Kubernetes resource.
With PMK, you can have your clusters deployed on-premises, in public clouds, or at the edge. In this document, we'll explain the steps on how to create an **on-premise [BareOS](https://platform9.com/docs/kubernetes/bareos-what-is-bareos) cluster** using **PMK** and connect it to **Microsoft Azure Arc**.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+## Prerequisites
+
+- An Ubuntu 20.04 installed physical machine or VM. For this demonstration, we would be using single node kubernetes cluster installation. This machine would also act as our management host.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:16]:
```suggestion
- An Ubuntu 20.04 installed physical machine or VM. In this scenario, we would be using single node kubernetes cluster installation. This machine would also act as our management host.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+  Once you have a successful login, create the Azure service principal (SP).
+
+  ```shell
+  az ad sp create-for-rbac -n "<service-principal-name>" --role contributor
+  ```
+
+  See below example of creating a service principal.
+
+  ![Screenshot of Creating Service Principal](./02.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:70]: Please align the service principle creation with the rest of the Jumpstart scenarios. See [example](https://github.com/microsoft/azure_arc/blob/main/docs/azure_arc_jumpstart/azure_arc_k8s/cluster_api/capi_azure/_index.md#prerequisites)

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+
+  ```shell
+  az group create -l <Azure Region> -n <resource group name>
+  ```
+
+  See below example of creating a resource group.
+  ![Screenshot of Creating Resource Group](./03.png)
+
+  You can verify it from the Azure portal.
+  ![Screenshot of Resource Group](./20.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:86]:
```suggestion
- Create an Azure resource group where you want the PMK cluster to show up.

  ```shell
  az group create -l <Azure Region> -n <resource group name>
  ```

  See below example of creating a resource group.
  Screenshot of Creating Resource Group

  You can verify it from the Azure portal.
  Screenshot of Resource Group
```

This should be part of the automation

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+
+  See below example of creating a service principal.
+
+  ![Screenshot of Creating Service Principal](./02.png)
+
+  > *Note: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a [specific Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).*
+
+  > **Note: Take a note of the *appId*, *password* and *tenant*, as that need to be set in an environment file later.**
+
+- Create an Azure resource group where you want the PMK cluster to show up.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:76]: This should be part of the automation

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+
+## Deployment
+
+This deployment consists of 3 sections.
+
+  1. Onboarding the node to Platform9 Management Plane.
+  2. Creating a PMK Cluster.
+  3. Connect the cluster to Microsoft Azure Arc.
+
+**Onboarding the node to Platform9 Management Plane**
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:96]:
```suggestion
### Onboarding the node to Platform9 Management Plane
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+
+- Login to your Management Plane.
+
+  ![Screenshot of PMK Management Plane Login Page](./04.png)
+
+  > **Note: If you do not have a registered Management Plane with Platform9, you can create one easily using [PMK Free Tier deployment](https://platform9.com/managed-kubernetes/)**
+
+- Onboard the Ubuntu host to your Management Plane.
+
+  For a BareOS cluster, you will need to have the nodes registered with the PMK Management Plane on which the cluster is to be deployed. For this, first add the node.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:106]:
```suggestion
  For a BareOS cluster, you will need to have the nodes registered with the PMK Management Plane on which the cluster is to be deployed. For this, first, add the node.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+
+  ![Screenshot of Onboarding a node](./05.png)
+
+  This should take you to the Node onboarding page. A **pf9ctl** utility is provided to setup the nodes and get connected with Management Plane.
+  Follow the instructions to download and install the utility, which we will use to prepare the node and connect it with your Platform9 Management Plane.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:111]:
```suggestion
  This should take you to the Node onboarding page. A **pf9ctl** utility is provided to set up the nodes and get connected with Management Plane.
  Follow the instructions to download and install the utility, which we will use to prepare the node and connect it with your Platform9 Management Plane.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+
+  ![Screenshot of PF9 CLI](./06.png)
+
+  > **Note: Preparing the node and connecting it to Management Plane might take approximately 4-5 minutes to complete.**
+
+
+**Creating a PMK Cluster**
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:118]:
```suggestion
### Creating a PMK Cluster
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+
+- The onboarded node should now be seen on our Management Plane. Now let's create a [PMK cluster](https://platform9.com/learn/learn/get-started-bare-metal) using that onboarded node.
+
+  The cluster creation is done from the PMK Management Plane UI. The steps for cluster creation will follow as below,
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:122]:
```suggestion
The onboarded node should now be seen on our Management Plane. To create a [PMK cluster](https://platform9.com/learn/learn/get-started-bare-metal) via the PMK Management Plane UI, follow the steps as described below.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 11]
```diff
@@ -0,0 +1,190 @@
...
+
+  - Click to add cluster to the Management Plane.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:124]:
```suggestion
  - Click to add a cluster to the Management Plane.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 12]
```diff
@@ -0,0 +1,190 @@
...
+
+  ![Screenshot of Creating One Click Cluster](./08.png)
+
+  - Enter the cluster name and k8s version. Submit the page and it will trigger the cluster creation on the node.
+
+  ![Screenshot of Cluster name entry](./09.png)
+
+  - The cluster should get created in a few minutes and the status should be reported as **Healthy**.
+
+  ![Screenshot of Cluster Created](./10.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:138]:
```suggestion
  ![Screenshot of Cluster Created](./10.png)
```

Would be good to add a red box around **Healthy**

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 13]
```diff
@@ -0,0 +1,190 @@
...
+
+**Connect the cluster to Microsoft Azure Arc**
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:140]:
```suggestion
### Connect the cluster to Microsoft Azure Arc**
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 14]
```diff
@@ -0,0 +1,190 @@
...
+
+- Clone the [azure_arc](https://github.com/deepuks/azure_arc) Jumpstart repository and change to the pf9 script directory.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:142]:
```suggestion
- Clone the [azure_arc](https://github.com/main/azure_arc) Jumpstart repository and change to the pf9 script directory.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 15]
```diff
@@ -0,0 +1,190 @@
...
+
+  ![Screenshot of Exporting values](./11.png)
+
+  > **Note: The values can referenced from the service principal and resource groups outputs.**
+
+- Set the KUBECONFIG variable for connecting to the K8s cluster. The kubeconfig file for the cluster is available for download from the Management Plane.
+
+  ![Screenshot to Download Kubeconfig](./13.png)
+
+  - An example output is shown below for setting the KUBECONFIG variable;
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:159]:
```suggestion
  An example output is shown below for setting the KUBECONFIG variable;
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 16]
```diff
@@ -0,0 +1,190 @@
...
+- Clone the [azure_arc](https://github.com/deepuks/azure_arc) Jumpstart repository and change to the pf9 script directory.
+
+  ```shell
+  git clone https://github.com/microsoft/azure_arc.git
+  cd azure_arc/azure_arc_k8s_jumpstart/pf9
+  ```
+
+- Set the variables according to your Azure service principal name and Azure environment in the *pf9_az.env* file.
+
+  ![Screenshot of Exporting values](./11.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:151]: Would be good to better mask values

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 17]
```diff
@@ -0,0 +1,190 @@
...
+
+- Set the KUBECONFIG variable for connecting to the K8s cluster. The kubeconfig file for the cluster is available for download from the Management Plane.
+
+  ![Screenshot to Download Kubeconfig](./13.png)
+
+  - An example output is shown below for setting the KUBECONFIG variable;
+
+  ![Screenshot to Export KUBECONFIG](./14.png)
+
+- Run the pf9_az_connect_k8s.sh script to connect the Platform9 Managed Kubernetes (PMK) cluster to Azure Arc.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:163]:
```suggestion
- Run the _pf9_az_connect_k8s.sh_ script to connect the Platform9 Managed Kubernetes (PMK) cluster to Azure Arc.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 18]
```diff
@@ -0,0 +1,190 @@
...
+
+  > **Note: The values can referenced from the service principal and resource groups outputs.**
+
+- Set the KUBECONFIG variable for connecting to the K8s cluster. The kubeconfig file for the cluster is available for download from the Management Plane.
+
+  ![Screenshot to Download Kubeconfig](./13.png)
+
+  - An example output is shown below for setting the KUBECONFIG variable;
+
+  ![Screenshot to Export KUBECONFIG](./14.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:161]: What this screenshot is suppose to show in the context of the guide?

**Review by likamrat — CHANGES_REQUESTED** [review]
- Add resource group creation to the automation
- Keep consistent png file's numbering. For example:
  - 11.png and then 13.png in the README
  - 03.png and then 20.png in the README
- 14.png - unclear, see comment
- markdown syntax
- Screenshots optimization

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 1 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: The changes have been commited.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 2 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: The changes have been commited.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 3 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Changes done.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 5 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Moved this to pf9_az_connect_k8s.sh script.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 6 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Corrected.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 7 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Corrected.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 8 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Done.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 9 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Corrected.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 10 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Done

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 11 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Corrected.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 12 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Image has been updated.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 13 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Done.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 14 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Changed the URL to https://github.com/microsoft/azure_arc

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 15 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Done.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 16 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Removed the pf9_az.env file and having the Azure env values exported into the pf9_az_connect_k8s.sh script.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 17 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: Done.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md (hunk 18 above)
Comment (deepuks) [reply docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md]: This has been removed and changed to include in shell.

**Comment by deepuks** [comment]
@likamrat 
The changes have been made with resource group creation moved into automation script and reducing the number of screenshots. Kindly review and let me know if we're good. Thanks.

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 19]
```diff
@@ -0,0 +1,179 @@
...
+title: "Platform9 Managed BareOS Kubernetes (PMK) cluster"
+linkTitle: "Platform9 Managed BareOS Kubernetes (PMK) cluster"
+weight: 1
+description: >
+---
+
+## Deploy a Platform9 Managed Kubernetes cluster and connect it to Azure Arc
+
+- The following document will guide on how to deploy a Kubernetes cluster with [Platform9 Managed Kubernetes (PMK)](https://platform9.com/managed-kubernetes/) and have it as a connected Azure Arc Kubernetes resource.
+- With PMK, you can have your clusters deployed on-premises, in public clouds, or at the edge. In this document, we'll explain the steps on how to create an **on-premise [BareOS](https://platform9.com/docs/kubernetes/bareos-what-is-bareos) cluster** using **PMK** and connect it to **Microsoft Azure Arc**.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:12]:
```suggestion
- The following document will guide on how to deploy a Kubernetes cluster with [Platform9 Managed Kubernetes (PMK)](https://platform9.com/managed-kubernetes/) and have it as a connected Azure Arc Kubernetes resource. With PMK, you can have your clusters deployed on-premises, in public clouds, or at the edge. In this document, we'll explain the steps on how to create an **on-premise [BareOS](https://platform9.com/docs/kubernetes/bareos-what-is-bareos) cluster** using **PMK** and connect it to **Microsoft Azure Arc**.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 20]
```diff
@@ -0,0 +1,179 @@
...
+  sudo apt-get install -y apt-transport-https ca-certificates curl
+  sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+  echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+  sudo apt-get update
+  sudo apt-get install -y kubectl
+  ```
+
+  > **Note:** The *kubectl* package is installed from Kubernetes repository, hence the Google Cloud public signing key need to be downloaded to enable the repository.
+
+  > **Note: All PMK cluster nodes would have kubectl installed once cluster is deployed.**
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:31]:
```suggestion
  > **Note: The *kubectl* package is installed from the Kubernetes repository, hence the Google Cloud public signing key need to be downloaded to enable the repository. Note: All PMK cluster nodes would have kubectl installed once the cluster is deployed.**
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 21]
```diff
@@ -0,0 +1,179 @@
...
+  cd azure_arc/azure_arc_k8s_jumpstart/pf9
+  ```
+
+- Set the variables according to your Azure service principal name and Azure environment in the *pf9_az_connect_k8s.sh* script.
+
+  ![Screenshot of Exporting values](./08.png)
+
+  > **Note: The values can referenced from the service principal output.**
+
+- Set the KUBECONFIG variable for connecting to the K8s cluster. The kubeconfig file for the cluster is available for download from the Management Plane.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:142]:
```suggestion
- Clone the [azure_arc](https://github.com/microsoft/azure_arc) Jumpstart repository and change to the pf9 script directory.

  ```shell
  git clone https://github.com/microsoft/azure_arc.git
  cd azure_arc/azure_arc_k8s_jumpstart/pf9
  ```

- Set the variables according to your Azure service principal name and Azure environment in the *pf9_az_connect_k8s.sh* script.

  Screenshot of Exporting values

  > **Note: The values can be referenced from the service principal output.**

- Set the KUBECONFIG variable for connecting to the K8s cluster. The kubeconfig file for the cluster is available for download from the Management Plane.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 22]
```diff
@@ -0,0 +1,179 @@
...
+  ```shell
+  git clone https://github.com/microsoft/azure_arc.git
+  cd azure_arc/azure_arc_k8s_jumpstart/pf9
+  ```
+
+- Set the variables according to your Azure service principal name and Azure environment in the *pf9_az_connect_k8s.sh* script.
+
+  ![Screenshot of Exporting values](./08.png)
+
+  > **Note: The values can referenced from the service principal output.**
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:140]:
```suggestion
  > **Note: The values can be referenced from the service principal output.**
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md [hunk 23]
```diff
@@ -0,0 +1,179 @@
...
+
+- Set the KUBECONFIG variable for connecting to the K8s cluster. The kubeconfig file for the cluster is available for download from the Management Plane.
+
+  ![Screenshot to Download Kubeconfig](./09.png)
+
+  - An example output is shown below for setting the _KUBECONFIG_ variable;
+
+  ```shell
+  export KUBECONFIG=<Enter the kubeconfig file path>
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md:149]:
```suggestion
  export KUBECONFIG=~/path/to/config/file
```

**Review by likamrat — CHANGES_REQUESTED** [review]
- Update 08.png to match the script. The image is missing the _azureRegion_ param

**Comment by deepuks** [comment]
> * Update 08.png to match the script. The image is missing the _azureRegion_ param

Updated the image.

Thanks.

**Review by likamrat — APPROVED** [review]
LGTM. Thank you for all the hard work @deepuks!

**Pull request merged**